src/web_interface/database_structure.py

```diff
@@ -32,14 +33,14 @@ def _create_tree_structure(list_of_dot_separated_strings):
     return structure_tree
 
 
-def _attach_field_to_tree(field, subtree):
-    splitted_field = field.split('.', 1)
-    if len(splitted_field) == 1:
+def _attach_field_to_tree(field: str, subtree):
```

Comment:
Why not type the subtree? It seems to be a (default)dict

Reply:
added some type hints